recipes/py-plumed/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "py-plumed" %}
+{% set version = "2.5.1" %}
+{% set sha256 = "de309980dcfd6f6e0e70e138856f4bd9eb4d8a513906a5e6389f18a5af7f2eba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plumed/plumed2/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

It looks like you're using `make`?
```suggestion
    - {{ compiler('cxx') }}
    - make
```